creme/creme_core/tests/forms/test_enumerable.py

```diff
@@ -29,7 +29,7 @@
 
 
 @override_settings(FORM_ENUMERABLE_LIMIT=100)
-class EnumerableChoiceSetTestCase(CremeTestCase):
+class FieldEnumerableChoiceSetTestCase(CremeTestCase):
```

Comment:
Tests for FieldEnumerableChoiceSet are covering EnumerableChoiceSet, so it is not really useful to test twice. Maybe later, I've done this separation to be able to create "fake" enumerables from registries or generated data instead of model fields.